### PR TITLE
Drop remaining Katello client configs

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/katello/katello-repos/katello-repos.spec [hunk 1]
```diff
@@ -28,35 +16,15 @@ Group:          Applications/Internet
...
 
-BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch:      noarch
 
 BuildRequires: sed
 
 %description
 Defines yum repositories for Katello and its sub projects, Candlepin and Pulp.
 
-%package -n katello-client-repos
```
Comment (ekohl) [inline packages/katello/katello-repos/katello-repos.spec]: This one also needs to be removed from comps

File: packages/katello/katello-repos/katello-repos.spec [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
-%endif
 
 %global prerelease .nightly
-%global release 4
+%global release 5
 
 Name:           katello-repos
 Version:        3.9.0
```
Comment (ekohl) [inline packages/katello/katello-repos/katello-repos.spec]: This one needs to be 3.10.0 now

**Comment by ekohl** [comment]
[test rpm]

**Comment by ehelms** [comment]
This should go in after https://github.com/theforeman/foreman-packaging/pull/3110

**Comment by ekohl** [comment]
Needs a rebase now due to conflicts

**Comment by ehelms** [comment]
@ekohl The updated changelog entry is there now that I was able to rebase it.

**Comment by ekohl** [comment]
[test rpm] because this is a new one:
```
Monday 22 October 2018  19:52:03 +0000 (0:00:00.128)       0:00:00.344 ******** 

wrapper script does not seem to be touching the log file in /var/lib/workspace/workspace/foreman-packaging-rpm-pr-test@tmp/durable-11c75ed1

(JENKINS-48300: if on a laggy filesystem, consider -Dorg.jenkinsci.plugins.durabletask.BourneShellScript.HEARTBEAT_CHECK_INTERVAL=300)

script returned exit code -1
```

**Pull request merged**